backend/discord_bot/tests/test_steward_commands.py

```
@@ -51,10 +50,16 @@ def _linked_user(self, name, discord_id, visible=True):
         )
         return user
 
-    def interaction(self, *, invoker=STEWARD_DISCORD_ID, interaction_id='inter-1', **options):
+    def interaction(self, *, invoker=AWARDER_DISCORD_ID, interaction_id='inter-1',
+                    channel_id=COMMUNITY_POINTS_CHANNEL_ID, channel_parent_id=None,
+                    roles=None, **options):
```

Ruff (0.16.1)

[warning] Missing type annotation for `**options`

(ANN003)